### DISCUSSION TITLE
Strange VS Code behaviour

### DISCUSSION THREAD

**Discussion by yonek** [community/community#76284] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hello,

I've encountered an issue with GitHub Copilot in Visual Studio Code and I'm seeking assistance to resolve it.

For some time, I have been using GitHub Copilot linked to my personal GitHub account, and it worked perfectly. However, last week, I decided to switch from my private subscription to a corporate subscription using my company's GitHub account. After logging into my editor with the new account, I noticed an unusual behavior.

The problem is that, a second after launching the editor, the Copilot icons from the sidebar disappear. Interestingly, Copilot itself continues to function and provides code suggestions as I type. However, the absence of these buttons completely prevents me from using the Copilot chat feature.

I am looking for advice or solutions on how to fix this issue. Has anyone else experienced something similar after switching subscriptions? Any insights or suggestions would be greatly appreciated.

Thank you in advance for your help.

**Comment by Akash1134** [comment]
Hi @yonek ,

That's strange. Have you tried signing out of the personal GitHub account, restart both IDE and Copilot, and then signing in with the corporate account so we know if its the account or IDE that's causing issues.

Thanks!

**Reply by yonek** [reply]
I did sign out my personal account, sign out/in my corporate account, reinstall VSC, reinstall plugins, restart PC.
